packages/eslint-plugin/src/rules/no-loss-of-precision.ts

```diff
@@ -0,0 +1,41 @@
+import { TSESTree } from '@typescript-eslint/experimental-utils';
+import baseRule from 'eslint/lib/rules/no-loss-of-precision';
```

Comment:
We need to guard against this not existing in <7.2.0, so that it only errors if a user attempts to actually use the rule in an old version of ESLint, and not if the user attempts to use the plugin.

This is why the integration test is failing.

Something like this (hack) will probably do:

```ts
import BaseRuleType from 'eslint/lib/rules/no-loss-of-precision';
import * as util from '../util';

const baseRule = ((): BaseRuleType | null => {
  try {
    return require('eslint/lib/rules/no-loss-of-precision');
  } catch {
    return null;
  }
}());

// ...
create(context) {
  if (baseRule == null) {
    throw new Error('@typescript-eslint/no-loss-of-precision requires at least ESLint v7.2.0');
  }
}
```

Reply:
@bradzacher  thanks for the solution. :)